components/RolesSection.tsx

```diff
@@ -0,0 +1,158 @@
+ roles={[]}
+ showLink={true}
+ columns={2} >
+          <Typography variant="bodyLarge" marginTop={'2rem'} textAlign={'center'}>All of our volunteers are vetted for experience, and sign a volunteer agreement before commencing work with Digital Aid Seattle.</Typography>
+  </RolesSection>
+
+* @2023 Digital Aid Seattle
+*/
+
+// icons for role cards
+import AccountTreeOutlinedIcon from '@mui/icons-material/AccountTreeOutlined'
+import ApartmentOutlinedIcon from '@mui/icons-material/ApartmentOutlined'
+import AutoStoriesOutlinedIcon from '@mui/icons-material/AutoStoriesOutlined'
+import BugReportOutlinedIcon from '@mui/icons-material/BugReportOutlined'
+import CampaignOutlinedIcon from '@mui/icons-material/CampaignOutlined'
+import CalculateOutlinedIcon from '@mui/icons-material/CalculateOutlined'
+import ChecklistOutlinedIcon from '@mui/icons-material/ChecklistOutlined';
+import CodeOutlinedIcon from '@mui/icons-material/CodeOutlined'
+import DescriptionOutlinedIcon from '@mui/icons-material/DescriptionOutlined'
+import Diversity3OutlinedIcon from '@mui/icons-material/Diversity3Outlined'
+import DrawOutlinedIcon from '@mui/icons-material/DrawOutlined'
+import EmojiPeopleOutlinedIcon from '@mui/icons-material/EmojiPeopleOutlined'
+import GavelRoundedIcon from '@mui/icons-material/GavelRounded'
+import HubOutlinedIcon from '@mui/icons-material/HubOutlined';
+import InsightsOutlinedIcon from '@mui/icons-material/InsightsOutlined'
+import IntegrationInstructionsOutlinedIcon from '@mui/icons-material/IntegrationInstructionsOutlined';
+import ManageAccountsOutlinedIcon from '@mui/icons-material/ManageAccountsOutlined'
+import PeopleOutlineOutlinedIcon from '@mui/icons-material/PeopleOutlineOutlined'
+import ScreenSearchDesktopOutlinedIcon from '@mui/icons-material/ScreenSearchDesktopOutlined'
+import ShareOutlinedIcon from '@mui/icons-material/ShareOutlined'
+import { Box, Button } from '@mui/material'
+import { ReactNode } from 'react'
+import { DASVolunteerRoleBasicInfo } from 'types'
+
+import ListItemWithIcon from './list/ListItemWithIcon'
+import { Section, Subheader } from './style-utils'
+
+// TODO: standardize roles between sanity and airtable
+const rolesMap = {
+  communityEngagementLiason: { role: 'community engagement liaison', icon: <CampaignOutlinedIcon />, },
+  dataAnalyst: { role: 'data analyst', icon: <ScreenSearchDesktopOutlinedIcon />, },
+  designer: { role: 'designer', icon: <DrawOutlinedIcon />, },
+  graphicDesigner: { role: 'graphic designer', icon: <DrawOutlinedIcon />, },
+  grantWriter: { role: 'grant writer', icon: <DescriptionOutlinedIcon />, },
+  legalHelp: { role: 'legal help', icon: <GavelRoundedIcon />, },
+  productManager: { role: 'product manager', icon: <Diversity3OutlinedIcon />, },
+  projectManager: { role: 'project manager', icon: <ManageAccountsOutlinedIcon />, },
+  uxResearcher: { role: 'user experience researcher', icon: <ScreenSearchDesktopOutlinedIcon />, },
```

Comment:
#53 and #54 look the same for me. I will contact you,  @wheninseattle , in private to clarify the requirements.